cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -242,7 +242,11 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         callAnalytics: function(event) {
             event.preventDefault();
             window.analytics.track('edx.bi.highlights.' + event.target.innerText.toLowerCase());
-            this.save(event);
+            if (event.target.className.indexOf('save') !== -1) {
+                this.save(event);
+            } else {
+                this.hide();
+            }
```

Comment:
Why was this change needed?

Reply:
There was a bug in the Highlights modal and in the new confirm modal where the model would be saved (and a POST request sent) even if you clicked "cancel" on the modal. It was because this function was bound to both "save" and "cancel".